### DISCUSSION TITLE
FLUX Tools will appear, should we expect it?

### DISCUSSION THREAD

**Discussion by SunGreen777** [vladmandic/sdnext#3589] [General]
Tell please, FLUX Tools will appear, should we expect it?

**Comment by vladmandic** [comment]
yes, its coming soon, i already spoke about it :)

**Comment by SAC020** [comment]
It seems to appear in the dev branch. 

I don't understand how to use the "Flux fill" for outpainting (I want to enlarge a portrait size photo laterally so it becomes a square photo). Can you please post a UI screenshot on how to perform this use case? Thank you

**Comment by SAC020** [comment]
Running out of VRAM (Nvidia 4090 24GB VRAM) trying to use Flux tools canny in img2img. Anything I can do?

**Reply by vladmandic** [reply]
what's your offloading set to?

**Reply by SAC020** [reply]
I am launching with --medvram and have the following settings

![image](https://github.com/user-attachments/assets/76589350-3fbd-4c81-85fd-c0d54322ac06)

Please let me know if to post additional settings which could be relevant to offloading

**Reply by vladmandic** [reply]
switch model offload mode to balanced.
offload=model is not enough for flux.

**Reply by SAC020** [reply]
> switch model offload mode to balanced. offload=model is not enough for flux.

Should I increase "max gpu memory for balanced offload mode" to >10GB, considering I have 24?

**Reply by vladmandic** [reply]
by default its set to 75% of available vram, so no idea why you have it at 10gb to start with.
so yes, increase to 18-20gb.

**Reply by SAC020** [reply]
Thanks, it works (barely - if I throw in a Lora on top it still crashes out of VRAM). Have you tried it yet? I got some bad outputs from it
